source/upgrade.txt

```diff
@@ -70,6 +70,12 @@ Version 6.x Breaking Changes
   ``tlsCertificateKeyFile`` connection options when you call the
   ``MongoClient.connect()`` method, not when you create the
   ``MongoClient`` instance.
+- The major version number of ``mongodb-client-encryption`` must now be the
+  same as the major version number of the {+driver-short+}. For example,
```

Comment:
Suggestion:

I think this could be more succinct and could emphasize the condition under which they need to match.

```suggestion
- If you add ``mongodb-client-encryption`` as a dependency,
   the major version number must match that of the {+driver-short}.
   For example,
```